tools/docs/manager.ts

```diff
@@ -56,6 +56,11 @@ sidebar_label: ${displayName}
         }
       }
       md += `For details on how to extend a manager's \`fileMatch\` value, please follow [this link](/modules/manager/#file-matching).\n\n`;
+      md += '## Supported datasources\n\n';
+      const escapedDatasources = (supportedDatasources || [])
+        .map((datasource) => `\`${datasource}\``)
+        .join(', ');
+      md += `This manager supports extracting the following datasources: ${escapedDatasources}.\n`;
```

Comment:
```suggestion
      md += `This manager supports extracting the following datasources: ${escapedDatasources}.\n\n`;
```

needs one more newline

We should later try to link those values